README.md

```diff
@@ -91,16 +91,14 @@ If you already have a local PostgreSQL installation, the easiest way to install
 Install packages on Ubuntu / Debian:
 
 ```bash
-curl https://install.citusdata.com/community/deb.sh > add-citus-repo.sh
-sudo bash add-citus-repo.sh
-sudo apt-get -y install postgresql-14-citus-11.0
+curl https://install.citusdata.com/community/deb.sh | sudo bash
```

Comment:
Note that we have different steps on https://www.citusdata.com/download/, and @gurkanindibay suggests we follow the same

Reply:
Some consider it bad practice to pipe shell script into sudo bash without first storing and reading it. If there's no space concern, I'd have a slight preference for the old format since it suggests some level of diligence (except the read step is missing) without loss of convenience.

Reply:
I think @gurkanindibay's point is that what we have in citusdata.com is simpler (1 line less). But, I think I'll follow your suggestion.

Reply:
@onderkalaci, does Marco's concern re piping into sudo bash apply to what's currently on the Citus download page & in the Citus Docs? And if so, should we talk to @begriffs about updating the Download Page's instructions (not urgently, but in the near future?)

Reply:
For Microsoft-related work, please ping @jonels-msft rather than my personal Github handle. :)